packages/blade/src/components/Typography/Heading/__tests__/Heading.native.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import renderWithTheme from '../../../../_helpers/testing/renderWithTheme.native';
+import Heading from '../';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<Heading />', () => {
+  it('should render Heading with variant "small"', () => {
+    const displayText = 'Get Started With Payment Gateway';
+    const { toJSON, getByText } = renderWithTheme(
+      <Heading type="normal" variant="small" weight="regular">
+        {displayText}
+      </Heading>,
+    );
+    expect(getByText('Get Started With Payment Gateway')).toBeTruthy();
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render Heading with variant "medium"', () => {
+    const displayText = 'Get Started With Payment Gateway';
+    const { toJSON, getByText } = renderWithTheme(
+      <Heading type="muted" variant="medium" weight="regular">
```

Comment:
Assuming all other tests for `type` prop is not added because we're only passing it down & BaseText takes care of it and it is tested there?

Reply:
yes